static/src/styles/landing/LandingUpdateFeed.styl

```diff
@@ -1,14 +1,34 @@
 .LandingUpdateFeed {
+    padding: grid(3);
```

Comment:
I'm going to leave some design nits here that we can likely follow up on later, where we can do a final pass over the entire design to make things nice and cohesive.

This has padding of 3 units but the introduction (with the hero) has a padding of 4 units, this is most obvious on mobile.